### ISSUE TITLE
Discharge battery

### ISSUE THREAD

**Issue by ReliableBob** [LilyGO/TTGO-T8-ESP32#17]
Hello to all.
I have a T8 V1.7 board.
I connected an 18650 lithium battery to the board. After turning off the main power, the board switched to battery power. After working for about eight hours, the power turned off. I thought it was ok, but I measured the voltage on the battery. It was 1.86 volts.
This is very small. The battery may be damaged ...
I studied the circuit and did not see the battery discharge control elements ... This is very bad ...
Correct if I'm wrong, but it turns out that there is no battery discharge control in such an excellent circuit? How can it be?

**Comment by tionebrr** [comment]
Yeah it is not such an excellent circuit. It is a cost optimized development board. Can't make coffee. 😁
